_data/toc.yaml

```diff
@@ -3773,24 +3796,12 @@ manuals:
         title: Overview
       - path: /ee/admin/backup/back-up-swarm/
         title: Back up Docker Swarm
-      - path: /ee/admin/backup/back-up-ucp/
-        title: Back up UCP
```

Comment:
Are these pages supposed to be removed from the ToC?

Reply:
Including the restores below

Reply:
Yes. Those are in two places in the current toc. I might not have made the right choice about which one to leave but my reasoning is that the topics are better under the component manual (vs under "manage Docker Enterprise" and "Restore Docker Enterprise"). It's kinda hard to see unless you have both areas of the ToC open at the same time.

Reply:
I agree with this decision. The UCP restore and back up links had been moved from the UCP section, but the Support team asked for it to be moved back there.  See https://github.com/docker/docker.github.io/pull/9649 for details.

Reply:
Great thank you Traci.